### PR TITLE
Add test question, question answer and test result models

### PR DESCRIPTION
**Pull request by lebedevaanastasiya**

### CODE REVIEW

File: database/models/testQuestions/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        type: Sequelize.TEXT,
+        field: 'question_text'
+    }
+}, {
+    freezeTableName: true,
+    createdAt: false,
+    updatedAt: false,
+});
+
+TestQuestion.hasMany(QuestionAnswer, { foreignKey: 'question_id' });
```
Comment (mbarouski) [inline database/models/testQuestions/index.js:21]: not `question_id`, but `test_id`, fk id on test table record

File: database/models/testResults/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    passDate: {
+        type: Sequelize.BIGINT,
+        field: 'pass_date'
+    }
+}, {
+    freezeTableName: true,
+    createdAt: false,
+    updatedAt: false
+});
+
```
Comment (mbarouski) [inline database/models/testResults/index.js:24]: add link to `test`

Comment (mbarouski) [reply database/models/testResults/index.js:24]: `belongsTo`

File: database/models/tests/index.js [hunk 3]
```diff
@@ -16,4 +18,7 @@ const Test = sequelize.define('test', {
     updatedAt: false,
 });
 
+Test.hasMany(TestQuestion, { foreignKey: 'test_id' });
+Test.hasMany(TestResult, {foreignKey: 'test_id'});
```
Comment (mbarouski) [inline database/models/tests/index.js]: delete

**Review by mbarouski — CHANGES_REQUESTED** [review]

File: database/models/testQuestions/index.js (hunk 1 above)
Comment (lebedevaanastasiya) [reply database/models/testQuestions/index.js:21]: Why test_id, if it is link from answer to question?

Comment (mbarouski) [reply database/models/testQuestions/index.js:21]: because we need to set name of field, by which we link answer with question

**Pull request merged**